templates/CRM/Admin/Page/Options.tpl

```diff
@@ -137,7 +137,7 @@
             <td class="crm-admin-options-description crm-editable" data-field="description" data-type="textarea">{if isset($row.description)}{$row.description}{/if}</td>
             <td class="nowrap crm-admin-options-order">{if isset($row.weight)}{$row.weight}{/if}</td>
             {if !empty($showIsDefault)}
-              <td class="crm-admin-options-is_default" align="center">{icon condition=$row.is_default}{ts}Default{/ts}{/icon}&nbsp;</td>
+              <td class="crm-admin-options-is_default" align="center">{if !empty($row.is_default)}{icon}{ts}Default{/ts}{/icon}{/if}&nbsp;</td>
```

Comment:
The `condition=$row.is_default` caused notices, and it's the same as surrounding the whole block in `{if}`